java/src/main/java/com/powsybl/python/network/NetworkModificationsCFunctions.java

```diff
@@ -226,6 +230,38 @@ public static PyPowsyblApiHeader.ArrayPointer<CIntPointer> getUnusedConnectableO
 
     }
 
+    @CEntryPoint(name = "createVoltageLevelTopology")
+    public static void createVoltageLevelTopology(IsolateThread thread, ObjectHandle networkHandle,
```

Comment:
we should not create a new method to get metadata each time we add a new method. it would be great if it can be refactored